distributed/deploy/cluster.py

```diff
@@ -122,7 +123,10 @@ def adapt(self, Adaptive=Adaptive, **kwargs) -> Adaptive:
         with ignoring(AttributeError):
             self._adaptive.stop()
         if not hasattr(self, "_adaptive_options"):
-            self._adaptive_options = {}
+            self._adaptive_options = dask.config.get("distributed.adaptive")
+            self._adaptive_options = {
+                k.replace("-", "_"): v for k, v in self._adaptive_options.items()
+            }
```

Comment:
I think that it would be best to apply these configuration defaults at the broadest possible point.  If someone calls `Adaptive` in some other way then we want to make sure that these configuration values are also applied, not only if they're called with the Cluster.adapt method.

I recommend setting the default values in the `Adaptive.__init__` constructor to None, and then doing a bunch of checks like the following:

```python
if interval is None:
    interval = dask.config.get("distributed.adaptive.interval")
...
```

This is similar to how other parts of the code base handle configuration options.

Reply:
That is sensible. Thanks for the implementation hint.